test/js/bun/test/test-test.test.ts

```diff
@@ -540,18 +540,18 @@ beforeEach: #2
 beforeEach: TEST-FILE
 beforeEach: one describe scope
 -- inside one describe scope --
+afterEach: one describe scope
+afterEach: TEST-FILE
 afterEach: #1
 afterEach: #2
-afterEach: TEST-FILE
-afterEach: one describe scope
 afterAll: one describe scope
 beforeEach: #1
 beforeEach: #2
 beforeEach: TEST-FILE
 -- the top-level test --
+afterEach: TEST-FILE
 afterEach: #1
 afterEach: #2
-afterEach: TEST-FILE
```

Comment:
@Jarred-Sumner 

> This continues to work with --preload adding global before{all,Each} and after{all,Each} which are cross-module, right?

Should be covered by this existing test case &minus; I've adjusted the output which I believed to be correct now, but PTAL.